plugins/CMakeLists.txt

```diff
@@ -71,6 +71,7 @@ if(BUILD_SUPPORTED)
     #dfhack_plugin(dwarfmonitor dwarfmonitor.cpp LINK_LIBRARIES lua)
     #add_subdirectory(embark-assistant)
     dfhack_plugin(eventful eventful.cpp LINK_LIBRARIES lua)
+    dfhack_plugin(export-map export-map.cpp COMPILE_FLAGS_GCC -fno-gnu-unique)
```

Comment:
what is requiring `-fno-gnu-unique`?

Reply:
probably a STL template; it can be hard to figure out which one because you have to pore over the libc++ implementation and that's very timeconsuming. 

`-fno-gnu-unique` should always be safe for DFHack plugins, as long as they don't crossload another another plug-in module

Reply:
Nothing is actually requiring this. Is what a (failed) attempt to make the plugin unload. I will have to check whether it unloads, now that I have removed the GDAL dependency. Keeping this there and the conversation open in the meantime.